Code/Project/Project/FileWritePage.xaml.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Forms;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Project
+{
+    /// <summary>
+    /// Interaction logic for FileWritePage.xaml
+    /// </summary>
+    public partial class FileWritePage : Page, IFileWritePage
+    {
+        public void NewFileWritePage()
+        {
+            if (null == Global.Window)
+            {
+                return;
+            }
+            InitializeComponent();
+
+            this.DataContext = this;
+            Global.Window.Content = this;
+        }
+
+        public int TaskCount { get { return Global.Tasks.Count; } }
+
+        private void TxtExport(object sender, RoutedEventArgs e)
+        {
+            var dialog = new FolderBrowserDialog();
+            DialogResult result = dialog.ShowDialog();
+
+            if(result == DialogResult.OK)
+            {
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".txt";
+                StreamWriter writer = new StreamWriter(path);
+
+                writer.WriteLine(DateTime.Now.ToString());
+                writer.WriteLine();
+                writer.WriteLine("Project: " + Global.ProjectName);
+                writer.WriteLine("SCRUM master: " + Global.ScrumMasterName);
+                writer.WriteLine("==================================");
+
+                foreach (Task task in Global.Tasks)
+                {
+                    writer.WriteLine(task.Name);
+                    writer.WriteLine(task.Description);
+                    writer.WriteLine("Story points: " + task.Rating);
+                    writer.WriteLine("==================================");
+                }
+
+                writer.WriteLine("Generated " + Global.Tasks.Count + " tasks");
+                writer.Close();
+            }
+        }
+
+        private void JsonExport(object sender, RoutedEventArgs e)
+        {
+
+            var dialog = new FolderBrowserDialog();
+            DialogResult result = dialog.ShowDialog();
+
+            if (result == DialogResult.OK)
+            {
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".json";

```

Comment:
The name would be better to include the date/time created AND the name of the project.

Reply:
sure